ee/clickhouse/sql/person.py

```diff
@@ -220,7 +220,7 @@
 """
 
 DELETE_PERSON_BY_ID = """
-INSERT INTO person (id, is_deleted) SELECT %(id)s, 1
+INSERT INTO person (id, team_id, is_deleted) SELECT %(id)s, %(team_id)s, 1
```

Comment:
Q: Is this consistent with how plugin server does it - if not, please sync it up!